docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -2952,11 +2954,11 @@ Create a volume
       "Name": "tardis",
       "Driver": "local",
       "Mountpoint": "/var/lib/docker/volumes/tardis",
-      "Status": null,
```

Comment:
w.r.t. the `Status` field, I think we can add some "fake" content, just to illustrate what it would look like. Change the "local" driver to something else to show that it's not the standard driver, and (if I read this line https://github.com/docker/docker/commit/36a1c56cf555f8fe9ceabeebb8fc956e05863fc7#diff-fbeef703d161e450a4a4eb21513ac9b0R114), the response could simply be, e.g.:

```json
"Status": { "Inside": "Bigger than the outside" }
```